lib/wcc/contentful/model/menu.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class WCC::Contentful::Model::Menu < WCC::Contentful::Model
+  validate_field :name, :String
+  validate_field :icon, :Asset, :optional
+  validate_field :rootButton, :Link, :optional, link_to: 'menuButton'
```

Comment:
thought we were going snake case?

Reply:
correct, my bad.  Though it should work regardless.